kani-driver/src/args/mod.rs

```diff
@@ -275,6 +275,10 @@ pub struct VerificationArgs {
     )]
     pub synthesize_loop_contracts: bool,
 
+    //Harness Output into individual files
+    #[arg(long, hide_short_help = true, requires("enable_unstable"))]
```

Comment:
I forgot to mention. We are deprecating `enable_unstable`, so can you please make sure the new feature can be enabled via `-Z unstable-options` instead?

Reply:
Hi there, is there any example of how to do that?

Reply:
Yes, unfortunately this cannot be done with clap directly. Here is an example:

https://github.com/model-checking/kani/blob/7ad4d1cc3109defea1131876f427f13599243ed8/kani-driver/src/args/mod.rs#L632-L640

In this case, you can use [`UnstableFeature::UnstableOptions`](https://github.com/model-checking/kani/blob/7ad4d1cc3109defea1131876f427f13599243ed8/kani_metadata/src/unstable.rs#L97).